team-01/src/main/java/ubc/cosc322/MonteCarloPlayer.java

```diff
@@ -130,5 +128,23 @@ public int getVisits() {
     public AmazonsAction getAction() {
       return action;
     }
+
+    // TODO
+    private double getUCB() {
+      // EXPLORATIONS = constant defined at the top of the class.
+      if (this.getVisits() == 0){
+        return EXPLORATIONS;
+      }
```

Comment:
The case of reaching an unvisited node should return infinity, not the exploration constant. To represent this value, you can use `Double.MAX_VALUE` instead.